sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_feature_set/feature_set_backfill_schema.py

```diff
@@ -0,0 +1,21 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from marshmallow import fields
+from azure.ai.ml._schema.core.fields import NestedField
+from azure.ai.ml._schema.core.schema import YamlFileSchema
+from azure.ai.ml._schema._feature_set.materialization_settings_schema import (
+    MaterializationComputeResourceSchema,
+)
+from azure.ai.ml._schema._feature_set.feature_window_schema import FeatureWindowSchema
+
+
+class FeatureSetBackfillSchema(YamlFileSchema):
+    name = fields.Str()
```

Comment:
name, version should be required

Reply:
user can specify name and version in the yml file.

Reply:
updated the schema to make name and version required.